web/manifest.webmanifest

```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "@NAME@",
-  "name": "@LONG_NAME@",
+  "short_name": "Worldview",
```

Comment:
I think we probably want the variables we just need to make sure they get translated properly during build

Reply:
Ok, made an update to `tasks/dist.js` to include  `build/worldview/manifest.webmanifest`

Reply:
Looks good, just need to put the vars back into the manifest

Reply:
Just pushed it up!